netbox/ipam/tests/test_filtersets.py

```diff
@@ -850,6 +857,15 @@ def test_site(self):
         params = {'site': [sites[0].slug, sites[1].slug]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 4)
 
+    def test_vlan_group(self):
+        vlan_groups = VLANGroup.objects.all()[:2]
+        params = {'vlan_group_id': [vlan_groups[0].pk, vlan_groups[1].pk]}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 4)
+        params = {'vlan_group': [vlan_groups[0].slug]}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
```

Comment:
These two lines can be removed; the following two lines provide sufficient test coverage.

```suggestion
        params = {'vlan_group': [vlan_groups[0].slug]}
        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
```